docs/topics/http/shortcuts.txt

```diff
@@ -107,8 +107,28 @@ This example is equivalent to::
    * An absolute or relative URL, which will be used as-is for the redirect
      location.
 
-   By default issues a temporary redirect; pass ``permanent=True`` to issue a
-   permanent redirect.
+   By default, this issues a temporary redirect (status code 302).
+
+   If ``permanent=True``, this issues a permanent redirect (status code 301).
+
+   If ``preserve_request_method=True``, the method and body of the original
+   request are reused to perform the redirected request. Temporary redirects
+   then have a 307 status code and permanent redirects have a 308 status code.
+
+   This is illustrated in the following table:
+
+   ================  =========  =======================
+   HTTP status code  permanent  preserve_request_method
+   ================  =========  =======================
+   301               ``True``   ``False``
+   302               ``False``  ``False``
+   307               ``False``  ``True``
+   308               ``True``   ``True``
+   ================  =========  =======================
```

Comment:
@sarahboyce @lorinkoz After a third iteration on these docs, I'm proposing to move the HTTP status code column to the end, so the left to right reading of this flows just like the function description does:

"when params have these values, the resulting HTTP status code is 30X"

```suggestion
   =========  ================ ================
   permanent  preserve_request HTTP status code
   =========  ================ ================
   ``True``   ``False``        301
   ``False``  ``False``        302
   ``False``  ``True``         307
   ``True``   ``True``         308
   =========  ================ ================
```